__tests__/bin.test.js

```diff
@@ -0,0 +1,28 @@
+const { exec } = require('child_process');
+const nodeVersion = require('parse-node-version')(process.version);
+
+describe('bin', () => {
+  if (nodeVersion.major > 12) {
```

Comment:
this also feels like another place we could leverage our `engines.node` field for, but I'm not as concerned about this.